### ISSUE TITLE
shreeyak - segmentation lab - give default value of batch size

### ISSUE THREAD

**Issue by Shreeyak** [udacity/robotics-beta-test-issues#674]
`steps_per_epoch` and `validation_steps` should be dependent on `batch_size`. Kindly provide a default batch size. 

Also mention a range of values to try out for `learning_rate` and `num_epochs`. I know the point is to experiment, but this model simply takes way to long to train. A broad range would also be fine, like say try a learning rate b/w 0.1 and 0.001.

**Comment by danzelmo** [comment]
We have decided to leave as is. This is something that can be experimented with.

**Comment by Shreeyak** [comment]
I feel some explanation of these terms is warranted, if students are going
to experiment on their own. Explain in minibatches, what batch_size,
steps_per_epoch do.

As we discussed in Slack, the meaning of an epoch itself has changed:
updates happen at every step instead of every epoch now. This information
should be conveyed, since that now affects the meaning of all these terms
as we've been taught.
 Also good to mention is that in typical applications, they are run for
20-50 epochs. Just as a reference.

**Comment by danzelmo** [comment]
Minibatches are covered in lesson 2, as well as a brief intro to stochastic gradient descent. The 20-50 epochs is not always the case. Imagine your dataset had 20million similar examples, then less then one epoch would be enough. We determine how long to train based on validation loss. You can look at the keras model api documentation https://keras.io/models/model/ search for fit_generator for an explanation of each of terms, we could potentially, add a link to this, in the notebook documentation.

**Comment by Shreeyak** [comment]
Haha! Exactly, see? The updates happening at the steps changes everything.
I hope you can understand how these parameters might not quite be clear to
the students and why it bears more explanation.

Also, to make this more evident, can the plots be drawn to have multiple
updates per epoch? Say every 10 or 20 or 50 steps, plot is updated, and
drawn together at the end of the epoch. This would demonstrate the concept
well.

**Comment by danzelmo** [comment]
Reclosing because this really is covered in the course material.
